1.8/accumulo_user_manual.html

```diff
@@ -3000,10 +3001,100 @@ <h3 id="_exporting_tables">6.11. Exporting Tables</h3>
 <p>Accumulo supports exporting tables for the purpose of copying tables to another
 cluster. Exporting and importing tables preserves the tables configuration,
 splits, and logical time. Tables are exported and then copied via the hadoop
-distcp command. To export a table, it must be offline and stay offline while
-discp runs. The reason it needs to stay offline is to prevent files from being
-deleted. A table can be cloned and the clone taken offline inorder to avoid
-losing access to the table. See <code>docs/examples/README.export</code> for an example.</p>
+<code>distcp</code> command. To export a table, it must be offline and stay offline while
+<code>distcp</code> runs. Staying offline prevents files from being deleted. An easy
```

Comment:
@mikewalch, nope. I missed that step :) I will have to update the asciidocs to match. Thanks for the info. I'll finish that up tomorrow and create a PR against Accumulo.

Reply:
No problem.  I moved the docs (starting with 2.0) from the Accumulo repo to the website to avoid the extra step. I wrote a [blog post](https://accumulo.apache.org/blog/2017/06/29/accumulo-documentation-improvements.html) about it.

Reply:
Hi @mikewalch Thanks for merging https://github.com/apache/accumulo/pull/350. 

Does this one still need to be merged as well?

Reply:
Yes I will merge it now